app/views/idv/welcome/show.html.erb

```diff
@@ -1,107 +1,61 @@
-            new_tab: true,
-          },
-          {
-            url: help_center_redirect_path(
+<% self.title = @title %>
+<%= render JavascriptRequiredComponent.new(
+      header: t('idv.welcome.no_js_header'),
+      intro: t('idv.welcome.no_js_intro', sp_name: @sp_name),
+      location: :idv_welcome,
+    ) do %>
+<%= render PageHeadingComponent.new.with_content(@title) %>
+  <p>
+    <%= t(
+          'doc_auth.info.getting_started_html',
+          sp_name: @sp_name,
+          link_html: new_tab_link_to(
+            t('doc_auth.info.getting_started_learn_more'),
+            help_center_redirect_path(
               category: 'verify-your-identity',
-              article: 'phone-number',
+              article: 'how-to-verify-your-identity',
               flow: :idv,
               step: :welcome,
-              location: 'missing_items',
+              location: 'intro_paragraph',
             ),
-            text: t('idv.troubleshooting.options.learn_more_address_verification_options'),
-            new_tab: true,
-          },
-          decorated_sp_session.sp_name && {
-            url: return_to_sp_failure_to_proof_url(step: 'welcome', location: 'missing_items'),
-            text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_sp_session.sp_name),
-            new_tab: true,
-          },
-        ].select(&:present?),
-      ) %>
-
-  <h3><%= t('doc_auth.instructions.privacy') %></h3>
-  <p>
-    <%= t('doc_auth.info.privacy', app_name: APP_NAME) %>
-  </p>
-  <p>
-    <%= new_tab_link_to(
-          t('doc_auth.instructions.learn_more'),
-          policy_redirect_url(flow: :idv, step: :welcome, location: :footer),
+          ),
         ) %>
   </p>
 
+  <h2><%= t('doc_auth.instructions.getting_started') %></h2>
+
+  <%= render ProcessListComponent.new(heading_level: :h3, class: 'margin-y-3') do |c| %>
+    <%= c.with_item(heading: t('doc_auth.instructions.bullet1')) do %>
+      <p><%= t('doc_auth.instructions.text1') %></p>
+    <% end %>
+    <%= c.with_item(heading: t('doc_auth.instructions.bullet2')) do %>
+      <p><%= t('doc_auth.instructions.text2') %></p>
+    <% end %>
+    <%= c.with_item(heading: t('doc_auth.instructions.bullet3')) do %>
+      <p><%= t('doc_auth.instructions.text3') %></p>
+    <% end %>
+    <%= c.with_item(heading: t('doc_auth.instructions.bullet4', app_name: APP_NAME)) do %>
+      <p><%= t('doc_auth.instructions.text4') %></p>
+    <% end %>
+  <% end %>
+<%= simple_form_for(
+      :doc_auth,
+      url: url_for,
+      method: 'put',
+      html: { autocomplete: 'off', class: 'margin-top-2 margin-bottom-5 js-consent-continue-form' },
+    ) do |f| %>
+
+  <div class="margin-top-4">
+    <%= render(
+          SpinnerButtonComponent.new(
+            type: :submit,
+            big: true,
+            wide: true,
+            spin_on_click: false,
+          ).with_content(t('doc_auth.buttons.continue')),
+        ) %>
+  </div>
```

Comment:
It seems like we shouldn't use a spinner button here if we don't intend for it to spin, but instead works as a normal submit button.

```suggestion
  <%= f.submit t('doc_auth.buttons.continue') %>
```